docs/root/configuration/observability/access_log/usage.rst

```diff
@@ -356,6 +356,9 @@ The following command operators are supported:
 %DOWNSTREAM_LOCAL_ADDRESS_WITHOUT_PORT%
     Same as **%DOWNSTREAM_LOCAL_ADDRESS%** excluding port if the address is an IP address.
 
+%CONNECTION_ID%
+  ID of the downstream connection.
```

Comment:
Please elaborate on what this value means and/or how it can be useful

Reply:
We have two uses:
- cross-reference multiple access log sinks
- cross-reference timer-based log reporters

Reply:
Updated.

Reply:
https://github.com/envoyproxy/envoy/issues/10936 will make it more useful in file access log context.

Reply:
Ok, that makes sense in the context of the other PR. Can you document the ID's uniqueness? (Unique within an execution of envoy, can duplicate between multiple envoys, can duplicate after envoy restarts)